src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -208,9 +260,15 @@
           When a tab displays a list, you can include the record count in the tab if that number will meaningfully impact user behavior.
```

Comment:
For modern theme, current guidelines for list pages is that tabs should not show counts

Reply:
What is the recommended fix here? We still have the `tabHeaderCount` property, so I think we need to update the guidance to differentiate between default theme and modern theme. It seems like we need to do at least a few things to make this work:
1. Update all of Adam's modern theme images to remove tab counts EXCEPT the image here for the **Count** section under **Options**.
2. Remove "Tab count" from the **Anatomy** list for modern theme.
3. Update the  **Count** section in modern theme to make it clear that while tab counts are technically an option, we strongly recommend against them. I'm thinking we want something similar to the **Button to view all options** section for lookup at https://developer.blackbaud.com/skyux-lookup/docs/lookup#section-options. It calls out a similar "optional-but-not-really-optional" setting. I think the new text could be something like ```While you technically have the option to display a record count when a tab displays a list, this option is only a available in default theme to support backwards compatibility. We strongly recommend against including record counts in tabs.```
4. Update the `tabHeaderCount` property description to similarly recommend against using the property in modern theme.
5. Update the demos to remove counts.

At least, those are the items that jump out at me. Any additional concerns or other thoughts?

Reply:
Last I talked with MG2 and Todd about this, it seems like we don't want to remove "counts" as a documented option for modern theme Tabs.

Reply:
Was waiting for @Blackbaud-ToddRoberts to reply, but so I don't forget, what does we mean by "current guidelines"? Do we have separate, unpublished guidelines somewhere? If so, what do we need to do to get them documented?

FWIW, as you can see from my list above, changing guidance even for something relatively minor like recommending NOT using something tab counts in modern when they are allowed in default theme will require a lot of docs work. And possibly dev work. I think in this case, the demos won't require a developer, but I'm not 100 percent sure on that.

Reply:
@blackbaud-johnly current guidelines for list page layout in the design section

Reply:
Thanks! I didn't make the connection to those list page guidelines.

Given that the list docs explicitly tell users not to use tab counts, I think we need to some update to the docs. Here are the 3 options I see:
1. Update the tab docs for modern theme as I described above.
2. Remove the tab label section from the list page docs.
3. Update the tab label section in the list page docs to explain that the guidance is specific to list pages but does not apply to tabs in other modern theme scenarios. (FWIW, I think this is the worst option because it creates case-by-case guidance that will almost certainly cause confusion and lead to inconsistency.)

@Blackbaud-AdamFunderburk and @Blackbaud-ToddRoberts, what do y'all think? If #1, I can think Adam can address the first point in this PR, and I can create a separate docs issue for the others.